documentation/handling_participants_moving_from_a_frozen_cohort.md

```diff
@@ -0,0 +1,32 @@
+# Handling participants moving from a cohort frozen for payments to the active cohort
+
+To enable the 2021 cohort to be frozen for payments, participants that had not completed their induction could be moved to the 2024 cohort by following these steps in order:
+
+1. A change happens to the participant such as changing from `withdrawn` to `active` status, transferring to another school or a change of mentor, that indicates they are continuing with their training
+2. We verify that the participant is eligible to move
+3. We move the participant
+
+## How do I determine whether the participant is eligible to move?
+
+In addition to seeing that they are continuing with training, we need to check the participant's induction status and declarations to determine whether they are eligible to move from 2021 to 2024:
+
+```ruby
+participant_profile.eligible_to_change_cohort_and_continue_training?(cohort:)
+```
+
+Where `cohort` is `Cohort.active_registration_cohort`
+
+## How do I move a 2021 participant to 2024?
+
+Once we are sure that a participant is eligible to be moved from 2021 to 2024 we use the `Induction::AmendParticipantCohort` service.  This performs lots of validations and safeguards that should not be skipped or performed manually.
+
+```ruby
+i = Induction::AmendParticipantCohort.new(participant_profile:,
+  																				source_cohort_start_year: 2021,
```

Comment:
The indentation here is out of bounds on the resulting MD doc.

Reply:
![tabs-spaces](https://github.com/DFE-Digital/early-careers-framework/assets/128088/56d10cc4-7c6c-4307-9f64-8d93625ffa6c)